components/script/dom/bindings/trace.rs

```diff
@@ -420,6 +421,13 @@ unsafe impl JSTraceable for Box<LayoutRPC + Send + 'static> {
     }
 }
 
+unsafe impl JSTraceable for Box<Runnable + Send + 'static> {
```

Comment:
This is not a safe construct, since it allows us to store non-GC safe values in Runnables.